bigbluebutton-html5/private/config/settings.yml

```
@@ -584,6 +584,8 @@ public:
     maxTypedAnswerLength: 45
     chatMessage: true
     quickPollConfirmationStep: true
+    quiz: true
+    correctAnswerMarker: "__"
```

I suggest make it clear that `correctAnswerMarker` is related with "quick poll" feature

```suggestion
    quiz:
      enabled: true
      quickPollCorrectAnswerSuffix: "__"
```